app/javascript/react/screens/App/Overview/components/Migrations/MigrationsCompletedList.js

```diff
@@ -235,7 +236,19 @@ const MigrationsCompletedList = ({
                           ),
                           !isMissingMapping && (
                             <ListView.InfoItem key={`${plan.id}-infraMappingName`}>
-                              {plan.infraMappingName}
+                              <a
+                                href="#"
+                                onClick={e => {
+                                  e.preventDefault();
+                                  filterMappingsListOnTransitionAction({
+                                    filterTypeId: 'name',
+                                    filterValue: plan.infraMappingName
+                                  });
+                                  redirectTo('/mappings');
```

Comment:
It looks like using the `redirectTo` helper is messing up the vert nav (i.e., the highlighted nav item is not correct after transition)